src/projects/index.ts

```diff
@@ -1,38 +1,8 @@
 import BaseApi from '../baseApi';
-import { RequestParams } from '../utils/params/requestParams';
-import { createRawApiResponse } from '../utils/response/rawApiResponse';
-import { createReadRawResponse, ReadRawResponse } from '../utils/response/readRawResponse';
-import { createResourceResponse } from '../utils/response/resourceResponse';
-import * as responseHandlers from '../utils/response/responseHandlers';
-import { Project } from './types';
 
-export class ProjectsEndpoint extends BaseApi {
-  public readonly resourceName = 'projects';
+const resourceName = 'projects';
+type ResourceName = typeof resourceName;
```

Comment:
no need for type as its only used once? If we want to have the types declared, i would do it in all endpoints.

Reply:
Will remove for projects